inference-engine/src/inference_engine/ie_network_reader.cpp

```diff
@@ -183,6 +183,7 @@ CNNNetwork details::ReadNetwork(const std::string& modelPath, const std::string&
         auto reader = it->second;
         // Check that reader supports the model
         if (reader->supportModel(modelStream)) {
+            modelStream.seekg(0, modelStream.beg);
```

Comment:
nit: Why cannot we add `seekg` to `ONNXReader::read`?

Reply:
I actually had it in the ONNX reader only in the beginning but moved it here to make sure that the stream always points to the beginning of the underlying file/string before the model is parsed. Should I move it to the ONNXReader?

Reply:
I think readers should control it.

Reply:
Ok, I'll do it in a separate PR though since all checks are finally green here.